src/frontend/helpers/library.ts

```diff
@@ -278,9 +274,14 @@ const updateGame = async (gameInfo: GameInfo) => {
     gameInfo?.channels?.[gameInfo.install.channelName ?? ''].license_config
       .tokens
   let siweValues = undefined
+  console.log(
```

Comment:
Do we want to change these to use the HP logger or something?

Reply:
definitely a debug log I forgot that I left in. just removed it